source/agora/network/Manager.d

```diff
@@ -786,6 +786,9 @@ public class NetworkManager
     /// Ditto
     private void addAddress (Address address)
     {
+        // Ensure we have a trailing slash for address
+        if (address[$ - 1] != '/')
+            address ~= '/';
```

Comment:
Hum, I'm not sure that'd work. Address could technically be an IP, couldn't it ?

Reply:
```
node-6_1  | 2021-09-16 08:44:11,239 Info [agora.network.Manager] - Found new FullNode: https://node-0:1735/
node-6_1  | 2021-09-16 08:44:11,240 Info [agora.network.Client] - Response of https://node-0:1735//register_listener : HTTP/1.1 404 Not Found
```

Reply:
The CI passed yet the clients failed to register. :thinking:

Reply:
> Hum, I'm not sure that'd work. Address could technically be an IP, couldn't it ?

Yes and we also add a slash in REST endpoints. I think it makes more sense to store in `Network.Manager` without the slash.